test/integration/chai/chai-policy-tests.cxx

```diff
@@ -13,14 +13,25 @@ static_assert(RAJA::detail::get_space<RAJA::omp_parallel_for_exec>::value == cha
 
 #if defined(RAJA_ENABLE_CUDA)
 static_assert(RAJA::detail::get_space<RAJA::cuda_exec<128> >::value == chai::GPU, "");
+#endif
+
+#if defined(RAJA_ENABLE_CUDA)
 static_assert(RAJA::detail::get_space<RAJA::IndexSet::ExecPolicy<RAJA::seq_segit, RAJA::cuda_exec<128> > >::value == chai::GPU, "");
```

Comment:
Is there a reason we are hard-coding block size to specific constants in this stuff?